bedrock/firefox/views.py

```diff
@@ -515,6 +522,21 @@ def get_template_names(self):
                 template = "firefox/developer/whatsnew.html"
             else:
                 template = "firefox/whatsnew/index.html"
+        elif version.startswith("119."):
+            if locale in ["de", "fr", "en-GB"] or country == "GB":
```

Comment:
Issue: If I'm in the UK, this means I always see this template irrespective of my language?

http://localhost:8000/pl/firefox/119.0/whatsnew/?geo=gb